apps/omg/lib/omg/state/transaction/payment.ex

```diff
@@ -124,8 +124,10 @@ defmodule OMG.State.Transaction.Payment do
   end
 
   defp reconstruct_outputs(outputs_rlp) do
+    # TODO: ugly but to be cleaned up in the abstract output PR soon, so leaving as is
```

Comment:
taking the liberty to cut a corner here. Handling this properly requires the abstract output type refactor, otherwise it's waste effort.